constructor/nsis/main.nsi.tmpl

```diff
@@ -692,44 +687,22 @@ Function .onInit
     Call mui_AnaCustomOptions_InitDefaults
     # Override custom options with explicitly given values from contruct.yaml.
     # If initialize_by_default (register_python_default) is None, do nothing.
-#if initialize_conda is True and initialize_by_default is True
+{%- if initialize_conda %}
+{%- if initialize_by_default %}
     ${If} $InstMode == ${JUST_ME}
         StrCpy $Ana_AddToPath_State ${BST_CHECKED}
     ${EndIF}
-#endif
-#if initialize_conda is True and initialize_by_default is False
+{%- else %}
     StrCpy $Ana_AddToPath_State ${BST_UNCHECKED}
-#endif
-#if register_python is True and register_python_default is True
-    StrCpy $Ana_RegisterSystemPython_State ${BST_CHECKED}
-#endif
-#if register_python is True and register_python_default is False
-    StrCpy $Ana_RegisterSystemPython_State ${BST_UNCHECKED}
-#endif
-#if check_path_length is True
-    StrCpy $CheckPathLength "1"
-#endif
-#if check_path_length is False
-    StrCpy $CheckPathLength "0"
-#endif
-#if keep_pkgs is True
-    StrCpy $Ana_ClearPkgCache_State ${BST_UNCHECKED}
-#endif
-#if keep_pkgs is False
-    StrCpy $Ana_ClearPkgCache_State ${BST_CHECKED}
-#endif
-#if pre_install_exists is True
-    StrCpy $Ana_PreInstall_State ${BST_CHECKED}
-#endif
-#if pre_install_exists is False
-    StrCpy $Ana_PreInstall_State ${BST_UNCHECKED}
-#endif
-#if post_install_exists is True
-    StrCpy $Ana_PostInstall_State ${BST_CHECKED}
-#endif
-#if post_install_exists is False
-    StrCpy $Ana_PostInstall_State ${BST_UNCHECKED}
-#endif
+{%- endif %}
+{%- endif %}
```

Comment:
Can these conditions be indented?

Reply:
Is NSIS whitespace sensitive? Only one way to know I guess 👀